lib/CrossTU/CrossTranslationUnit.cpp

```diff
@@ -199,10 +199,10 @@ CrossTranslationUnitContext::getCrossTUDefinition(const FunctionDecl *FD,
                                                   StringRef CrossTUDir,
                                                   StringRef IndexName,
                                                   bool DisplayCTUProgress) {
```

Comment:
Perhaps we should add `assert(FD);` as the first line of `getCrossTUDefinition`.

Reply:
Ok, I'll redo this PR.